domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/PerformanceMetricsController.kt

```diff
@@ -114,6 +115,7 @@ class PerformanceMetricsController @Inject constructor(
       this.isAppInForeground = this@PerformanceMetricsController.isAppInForeground
       this.storageTier = performanceMetricsAssessor.getDeviceStorageTier()
       this.memoryTier = performanceMetricsAssessor.getDeviceMemoryTier()
+      this.networkType = networkConnectionUtil.getCurrentConnectionStatus().toNetworkType()
```

Comment:
Nice catch--is this also covered in tests? Would be nice to make sure this correctly verified as present, and calculated correctly in relevant situations.

Reply:
added additional tests to cover these scenarios.